### PR TITLE
Connect ootp responses to slack output.

### PR DESCRIPTION
**Pull request by jbeder**

### CODE REVIEW

File: src/bin/slackApi.ts [hunk 1]
```diff
@@ -71,8 +71,8 @@ async function getTextInternal(aiClient: AIClient, channel, input, reminders) {
 
 const SUPER_CLUSTER_USER_STRING = 'UVBBEEC4A';
 
-async function sendOotpChat(messages) {
-  axios
+async function sendOotpChat(messages, channel, say) {
+  return axios
```
Comment (dan-perron) [inline src/bin/slackApi.ts]:
```suggestion
  const response = await axios
```

Comment (dan-perron) [reply src/bin/slackApi.ts]: I think you can remove the promise chains and just use `await` to make this a lot cleaner

Comment (jbeder) [reply src/bin/slackApi.ts]: Done. I always forget about async/await.

File: src/bin/slackApi.ts [hunk 2]
```diff
@@ -71,26 +71,39 @@ async function getTextInternal(aiClient: AIClient, channel, input, reminders) {
...
-    .then((response) => {
-      console.log('chat response', response.data);
-    })
-    .catch((error) => {
-      console.log('chat error', error);
-    });
+      messages,
+    },
+    {
+      headers: { 'Content-Type': 'application/json' },
```
Comment (dan-perron) [inline src/bin/slackApi.ts]: Might be able to drop this?

File: src/bin/slackApi.ts [hunk 3]
```diff
@@ -71,26 +71,39 @@ async function getTextInternal(aiClient: AIClient, channel, input, reminders) {
...
+    }
+  );
+  const data = response.data;
```
Comment (dan-perron) [inline src/bin/slackApi.ts]:
```suggestion
  const {data} = response;
```
Is more idiomatic

**Review by dan-perron — APPROVED** [review]

File: src/bin/slackApi.ts (hunk 2 above)
Comment (jbeder) [reply src/bin/slackApi.ts]: I think you're right, per https://axios-http.com/docs/post_example.

**Pull request merged**